crates/optimism/node/src/node.rs

```diff
@@ -556,35 +556,20 @@ impl<NetworkT> OpAddOnsBuilder<NetworkT> {
 }
 
 /// A regular optimism evm and executor builder.
-#[derive(Debug, Copy)]
+#[derive(Debug, Copy, Clone, Default)]
 #[non_exhaustive]
-pub struct OpExecutorBuilder<ChainSpec = OpChainSpec, Primitives = OpPrimitives> {
-    /// Marker for chain spec type.
-    _cs: PhantomData<ChainSpec>,
-    /// Marker for primitives type.
-    _p: PhantomData<Primitives>,
-}
-
-impl<ChainSpec, Primitives> Clone for OpExecutorBuilder<ChainSpec, Primitives> {
-    fn clone(&self) -> Self {
-        Self::default()
-    }
-}
-
-impl<ChainSpec, Primitives> Default for OpExecutorBuilder<ChainSpec, Primitives> {
-    fn default() -> Self {
-        Self { _cs: PhantomData, _p: PhantomData }
-    }
-}
+pub struct OpExecutorBuilder;
 
-impl<Node, ChainSpec, Primitives> ExecutorBuilder<Node> for OpExecutorBuilder<ChainSpec, Primitives>
+impl<Node> ExecutorBuilder<Node> for OpExecutorBuilder
 where
-    Node: FullNodeTypes<Types: NodeTypes<ChainSpec = ChainSpec, Primitives = Primitives>>,
-    ChainSpec: EthChainSpec + OpHardforks,
-    Primitives: NodePrimitives,
-    OpEvmConfig<ChainSpec, Primitives>: ConfigureEvm<Primitives = Primitives> + 'static,
+    Node: FullNodeTypes<Types: NodeTypes>,
+    <Node::Types as NodeTypes>::ChainSpec: EthChainSpec + OpHardforks,
```

Comment:
Should we let `EthChainSpec` or just 
```suggestion
    <Node::Types as NodeTypes>::ChainSpec: OpHardforks,
```

Reply:
just `OpHardforks` will work